pthread_setname_np is a glibc extension not defined in POSIX
and is not supported by musl libc.
--- a/privileged.c
+++ b/privileged.c
@@ -920,7 +920,7 @@
 # endif
 	} else {
 		register_blockthread();
-		pthread_setname_np(pthread_self(), "clsync-helper");
+		//pthread_setname_np(pthread_self(), "clsync-helper");
 	}
 	cap_drop(ctx_p, ctx_p->caps);
 
